docs/content/en/docs/getting-started/ports/_index.md

```diff
@@ -84,7 +111,7 @@ The following table displays ports that need to be accessible from the VMware pr
 | TCP      | Inbound   | 2379        | Manager                 | Etcd API endpoint       |
 | TCP      | Inbound   | 2380        | Manager                 | Etcd API endpoint       |
 
-## Nutanix provider
+##### Nutanix provider
```

Comment:
```suggestion
### Nutanix provider
```
I suggested changing the heading above to ##. If you agree, then this should be ###.